docker/docker-compose.yml

```diff
@@ -0,0 +1,209 @@
+      dockerfile: Dockerfile
+    environment:
+      LEDGER_URL: ${LEDGER_URL}
+      APPLICATION_URL: ${APPLICATION_URL:-http://localhost:5001}
+      ENDPOINT_URL: ${ENDPOINT_URL:-http://localhost:5001}
+      CONFIG_ROOT: ../config
+      ENVIRONMENT: ${ENVIRONMENT:-default}
+      ENABLE_GUNICORN: ${ENABLE_GUNICORN:-0}
+      INDY_LEDGER_URL: ${LEDGER_URL:-http://localhost:9000}
+      FORMS: ${FORMS:-}
+      ISSUERS: ${ISSUERS:-}
+      LOG_LEVEL: ${LOG_LEVEL:-}
+      PYTHON_ENV: ${PYTHON_ENV:-development}
+      TOB_API_URL: ${TOB_API_URL:-}
+      TOB_APP_URL: ${TOB_APP_URL:-}
+      WALLET_TYPE: ${WALLET_TYPE}
+      POSTGRESQL_WALLET_HOST: ${POSTGRESQL_WALLET_HOST}
+      POSTGRESQL_WALLET_PORT: ${POSTGRESQL_WALLET_PORT}
+      POSTGRESQL_WALLET_USER: ${POSTGRESQL_USER}
+      POSTGRESQL_WALLET_PASSWORD: ${POSTGRESQL_PASSWORD}
+      POSTGRESQL_WALLET_ADMIN_PASSWORD: ${POSTGRESQL_ADMIN_PASSWORD}
+      WALLET_ENCRYPTION_KEY: ${WALLET_ENCRYPTION_KEY}
+      WALLET_SEED_VONX: ${WALLET_SEED_VONX:-}
+      ISSUER_WALLET_DID: ${ISSUER_WALLET_DID}
+      HTTP_FORCE_CLOSE_CONNECTIONS: "true"
+      AGENT_ADMIN_URL: http://myorg-agent:${AGENT_ADMIN_PORT}
+      TOB_AGENT_ADMIN_URL: http://tob-agent:${TOB_AGENT_ADMIN_PORT}
+      TOB_CONNECTION_NAME: tob-agent
+      WEBHOOK_PORT: ${WEBHOOK_PORT}
+    networks:
+      - orgbook
+      - myorg
+    ports:
+      - ${WEBHOOK_PORT}:${WEBHOOK_PORT}
+    depends_on:
+      - myorg-agent
+      - myorg-wallet-db
+    entrypoint: >
+      /bin/bash -c "
+        echo sleeping ...;
+        sleep 15;
+        ../docker-entrypoint.sh;"
+
+  myorg-agent:
+    image: tob-agent
+    environment:
+      - LEDGER_URL=${LEDGER_URL}
+      - WALLET_TYPE=${WALLET_TYPE}
+      - POSTGRESQL_WALLET_HOST=${POSTGRESQL_WALLET_HOST}
+      - POSTGRESQL_WALLET_PORT=${POSTGRESQL_WALLET_PORT}
+      - POSTGRESQL_WALLET_USER=${POSTGRESQL_USER}
+      - POSTGRESQL_WALLET_PASSWORD=${POSTGRESQL_PASSWORD}
+      - POSTGRESQL_WALLET_ADMIN_PASSWORD=${POSTGRESQL_ADMIN_PASSWORD}
+      - WALLET_ENCRYPTION_KEY=${WALLET_ENCRYPTION_KEY}
+      - WALLET_SEED_VONX=${WALLET_SEED_VONX:-}
+      - ISSUER_WALLET_DID=${ISSUER_WALLET_DID}
+      - WEBHOOK_URL=${WEBHOOK_HOST}/api/agentcb
+      - AGENT_IN_PORT_1=${AGENT_IN_PORT_1}
+      - AGENT_IN_PORT_2=${AGENT_IN_PORT_2}
+      - AGENT_IN_PORT_3=${AGENT_IN_PORT_3}
+      - AGENT_ADMIN_PORT=${AGENT_ADMIN_PORT}
+      - AGENT_NAME=${AGENT_NAME}
+    networks:
+      - orgbook
+      - myorg
+    ports:
+      - ${AGENT_IN_PORT_1}:${AGENT_IN_PORT_1}
+      - ${AGENT_IN_PORT_2}:${AGENT_IN_PORT_2}
+      - ${AGENT_IN_PORT_3}:${AGENT_IN_PORT_3}
+      - ${AGENT_ADMIN_PORT}:${AGENT_ADMIN_PORT}
+    depends_on:
+      - myorg-wallet-db
+    entrypoint: /bin/bash
+    command: [
+        "-c",
+        " sleep 5; \
+        aca-py \
+        --inbound-transport http '0.0.0.0' ${AGENT_IN_PORT_1} \
+        --inbound-transport http '0.0.0.0' ${AGENT_IN_PORT_2} \
```

Comment:
There is an example of running two separate inbound transports of the same type, but isn't really necessary in this situation. This line can be removed.